fastlane/helpers/github.rb

```diff
@@ -0,0 +1,10 @@
+def create_release_management_pull_request(base_branch, title)
+  create_pull_request(
+      api_token: ENV['GITHUB_TOKEN'],
+      repo: GHHELPER_REPO,
+      title: title,
+      head: Fastlane::Helper::GitHelper.current_git_branch,
+      base: base_branch,
+      labels: 'Releases'
```

Comment:
We can add `reviewers` field, but I wanted to skip that for now to not create extra noise. I have tested creating PRs a few times and will likely do a few more. Once everything settles down, it's easy enough to add this field.